libraries/matrix/api/src/main/kotlin/io/element/android/libraries/matrix/api/roomlist/RoomSummary.kt

```diff
@@ -36,13 +36,15 @@ sealed interface RoomSummary {
 data class RoomSummaryDetails(
     val roomId: RoomId,
     val name: String,
-    val canonicalAlias: String? = null,
+    val canonicalAlias: String?,
     val isDirect: Boolean,
-    val avatarURLString: String?,
+    val avatarUrl: String?,
     val lastMessage: RoomMessage?,
-    val lastMessageTimestamp: Long?,
-    val unreadNotificationCount: Int,
-    val inviter: RoomMember? = null,
-    val notificationMode: RoomNotificationMode? = null,
-    val hasOngoingCall: Boolean = false,
-)
+    val numUnreadMessages: Int,
```

Comment:
I'd prefer a clear `numberOfUnreadMessages` and `numberOfUnreadMontions` name, but up to you

Reply:
Yes, always the same debate if we want to stick with the SDK name...